tests/integration_test/run_int_test_locally.sh

```diff
@@ -16,15 +16,15 @@ VAULT_TOKEN_PATH=${VAULT_TOKEN_PATH:-"${HOME}/.vault-token"}
 # TENX_MODE=${11}
 # TENX_VERSION=${12}
 # TENX_DIR=${13}
-# OPTIMUS_MODE=${14}
-# OPTIMUS_VERSION=${15}
-# OPTIMUS_DIR=${16}
-# SS2_MODE=${17}
-# SS2_VERSION=${18}
-# SS2_DIR=${19}
-# TENX_SUBSCRIPTION_ID=${20:-"placeholder_10x_subscription_id"}
-# SS2_SUBSCRIPTION_ID=${21:-"placeholder_ss2_subscription_id"}
-# OPTIMUS_SUBSCRIPTION_ID=${22:-"placeholder_optimus_subscription_id"}
+# TENX_SUBSCRIPTION_ID=${14:-"placeholder_10x_subscription_id"}
+# OPTIMUS_MODE=${15}
+# OPTIMUS_VERSION=${16}
+# OPTIMUS_DIR=${17}
+# OPTIMUS_SUBSCRIPTION_ID=${18:-"placeholder_optimus_subscription_id"}
+# SS2_MODE=${19}
+# SS2_VERSION=${20}
+# SS2_DIR=${21}
+# SS2_SUBSCRIPTION_ID=${22:-"placeholder_ss2_subscription_id"}
```

Comment:
I'm getting the value `SS2_SUBSCRIPTION_ID=placeholder_ss2_subscription_id placeholder_optimus_subscription_id` when I run this test locally 🤔

Reply:
Oh, it's actually `"placeholder_10x_subscription_id placeholder_optimus_subscription_id placeholder_ss2_subscription_id"` for all of the subscription ids. I think adding the delimiter `-d " "` to the cut command at the end will fix it